common/firmware/STM32G491RETx/STM32G491RETx_FLASH.ld

```diff
@@ -59,6 +59,27 @@ SECTIONS
     KEEP(*(.fw_update_flag_section))
   } > RAM
 
+  _siccmram = LOADADDR(.ccmram);
+  
+  /* CCM-RAM section 
+  * 
+  * IMPORTANT NOTE! 
+  * If initialized variables will be placed in this section,
+  * the startup code needs to be modified to copy the init-values.  
+  */
+  .ccmram :
+  {
+    . = ALIGN(4);
+    _sccmram = .;       /* create a global symbol at ccmram start */
+    *(.ccmram)
+    *(.ccmram*)
+    *(.text.*motor_handler*) 
```

Comment:
I think this relies on `-ffunction-sections` being on so let's drop a note wherever we're enabling this in cmake

Reply:
That's set in the `cmake` subtree, can either add it in a PR there or drop something in the linker file